dot/parachain/backing/candidate_backing.go

```diff
@@ -1,121 +1,664 @@
+		}
+
+		ourIndex := rpState.TableContext.validator.index
+		if signedStatementWithPVD.SignedFullStatement.ValidatorIndex == ourIndex {
+			return nil
+		}
+
+		if rpState.AwaitingValidation[summery.Candidate] {
+			// Job already running
+			attesting.backing = append(attesting.backing, signedStatementWithPVD.SignedFullStatement.ValidatorIndex)
+			return nil
+		}
+		// No job, so start another with current validator:
+		attesting.fromValidator = signedStatementWithPVD.SignedFullStatement.ValidatorIndex
+
+	default:
+		return fmt.Errorf("invalid statementVDT index: %d", statementVDT.Index())
+	}
+
+	// After `import_statement` succeeds, the candidate entry is guaranteed to exist.
+	if pc, ok := cb.perCandidate[summery.Candidate]; ok {
+		rpState.kickOffValidationWork(
+			cb.SubSystemToOverseer,
+			chRelayParentAndCommand,
+			pc.persistedValidationData,
+			attesting,
+		)
+	}
+
+	return nil
 }
 
-// SignedFullStatementWithPVD represents a signed full statement along with associated Persisted Validation Data (PVD).
-type SignedFullStatementWithPVD struct {
-	SignedFullStatement     parachaintypes.UncheckedSignedFullStatement
+func (rpState *perRelayParentState) importStatement(
+	subSystemToOverseer chan<- any,
+	signedStatementWithPVD SignedFullStatementWithPVD,
+	perCandidate map[parachaintypes.CandidateHash]perCandidateState,
+) (*Summary, error) {
+	statementVDT, err := signedStatementWithPVD.SignedFullStatement.Payload.Value()
+	if err != nil {
+		return nil, fmt.Errorf("getting value from statementVDT: %w", err)
+	}
+
+	if statementVDT.Index() == 2 { // Valid
+		return rpState.Table.importStatement(&rpState.TableContext, signedStatementWithPVD)
+	}
+
+	// PersistedValidationData should not be nil if the statementVDT is Seconded.
+	if signedStatementWithPVD.PersistedValidationData == nil {
+		return nil, fmt.Errorf("persisted validation data is nil")
+	}
+
+	statementVDTSeconded := statementVDT.(parachaintypes.Seconded)
+	candidateHash := parachaintypes.CandidateHash{
+		Value: common.MustBlake2bHash(scale.MustMarshal(statementVDTSeconded)),
+	}
+
+	if _, ok := perCandidate[candidateHash]; ok {
+		return rpState.Table.importStatement(&rpState.TableContext, signedStatementWithPVD)
+	}
+
+	if rpState.ProspectiveParachainsMode.IsEnabled {
+		chIntroduceCandidate := make(chan error)
+		subSystemToOverseer <- ProspectiveParachainsMessage{
+			Value: IntroduceCandidate{
+				IntroduceCandidateRequest: IntroduceCandidateRequest{
+					CandidateParaID:           parachaintypes.ParaID(statementVDTSeconded.Descriptor.ParaID),
+					CommittedCandidateReceipt: parachaintypes.CommittedCandidateReceipt(statementVDTSeconded),
+					PersistedValidationData:   *signedStatementWithPVD.PersistedValidationData,
+				},
+				Ch: chIntroduceCandidate,
+			},
+		}
+
+		introduceCandidateErr := <-chIntroduceCandidate
+		if introduceCandidateErr != nil {
+			return nil, fmt.Errorf("%w: %w", ErrRejectedByProspectiveParachains, introduceCandidateErr)
+		}
+
```

Comment:
Add a comment, create an issue for prospective parachain and handle it on overseer subsystem
```suggestion

		// TODO: Handle this message in overseer.
		// That is forward to the Prospective Parachains Subsystem.
```